examples/main/views/custom-interaction.js

```diff
@@ -62,7 +64,7 @@ const propTypes = {
   height: PropTypes.number.isRequired
 };
 
-export default class TiltExample extends Component {
+export default class Example extends Component {
```

Comment:
Is there a reason this file is called custom-interaction? It's not using a custom controller class.